test/functional/tools/galaxy_slots_and_memory.xml

```diff
@@ -1,6 +1,6 @@
 <tool id="galaxy_slots_and_memory" name="Galaxy Slots and Memory" version="1.0.0" profile="22.05">
     <command><![CDATA[
-echo "\$GALAXY_SLOTS\n\$GALAXY_MEMORY_MB_PER_SLOT\n\$GALAXY_MEMORY_MB" > '$output'
+echo "\$GALAXY_SLOTS\n$GALAXY_MEMORY_MB_PER_SLOT\n$GALAXY_MEMORY_MB" > '$output'
```

Comment:
```suggestion
echo "$GALAXY_SLOTS\n$GALAXY_MEMORY_MB_PER_SLOT\n$GALAXY_MEMORY_MB" > '$output'
```

I suppose?

Reply:
No, that would make cheetah think there's a variable called `GALAXY_SLOTS\n$GALAXY_MEMORY_MB_PER_SLOT\n$GALAXY_MEMORY_MB`

Reply:
Ah, sorry, then I think the correct fix is:

```suggestion
echo -e "\$GALAXY_SLOTS\n\$GALAXY_MEMORY_MB_PER_SLOT\n\$GALAXY_MEMORY_MB" > '$output'
```